.travis.yml

```diff
@@ -6,10 +6,20 @@ node_js:
 - 8 # EOL: December 2019
 - 6 # EOL: April 2019
 
+os:
+  - linux
```

Comment:
I didn't include osx/windows for all node versions because these are indeed slower. osx is practically linux, so no reason to build it multiple times, and windows we didn't build multi versions on appveyor either.